tsconfig.json

```diff
@@ -26,6 +24,6 @@
   "ts-node": {
     "compilerOptions": {}
   },
-
-  "exclude": ["node_modules", "babel.config.js", "metro.config.js", "jest.config.js", "android"]
+  "exclude": ["node_modules", "babel.config.js", "metro.config.js", "jest.config.js", "android"],
+  "extends": "expo/tsconfig.base"
```

Comment:
This `extends` is what was failing tests in the SDK 50 branch, the `jest` runne/`ts-node` wasn't able to see the expo module's config.

Reply:
Hm - removing this from the `tsconfig.json` and `ts-node` from `package.json` still nets me an error.

```diff
$ git diff
diff --git a/package.json b/package.json
index 029ce4b..b2785bc 100644
--- a/package.json
+++ b/package.json
@@ -174,7 +174,6 @@
     "react-native-svg-transformer": "^1.3.0",
     "react-test-renderer": "^18.2.0",
     "timezone-mock": "^1.3.6",
-    "ts-node": "^10.9.2",
     "ts-to-zod": "^3.1.2",
     "typescript": "^5.3.0"
   },
diff --git a/tsconfig.json b/tsconfig.json
index 4b93570..a665fed 100644
--- a/tsconfig.json
+++ b/tsconfig.json
@@ -24,6 +24,5 @@
   "ts-node": {
     "compilerOptions": {}
   },
-  "exclude": ["node_modules", "babel.config.js", "metro.config.js", "jest.config.js", "android"],
-  "extends": "expo/tsconfig.base"
+  "exclude": ["node_modules", "babel.config.js", "metro.config.js", "jest.config.js", "android"]
 }
```

```
$ jest
Error: Jest: Failed to parse the TypeScript config file /home/stevekuznetsov/code/nwacus/avy/src/github.com/nwacus/avy/jest.config.ts
  Error: Jest: 'ts-node' is required for the TypeScript configuration files. Make sure it is installed
Error: Cannot find package 'ts-node' imported from /home/stevekuznetsov/code/nwacus/avy/src/github.com/nwacus/avy/node_modules/jest-config/build/readConfigFileAndSetRootDir.js
    at readConfigFileAndSetRootDir (/home/stevekuznetsov/code/nwacus/avy/src/github.com/nwacus/avy/node_modules/jest-config/build/readConfigFileAndSetRootDir.js:116:13)
    at async readInitialOptions (/home/stevekuznetsov/code/nwacus/avy/src/github.com/nwacus/avy/node_modules/jest-config/build/index.js:403:13)
    at async readConfig (/home/stevekuznetsov/code/nwacus/avy/src/github.com/nwacus/avy/node_modules/jest-config/build/index.js:147:48)
    at async readConfigs (/home/stevekuznetsov/code/nwacus/avy/src/github.com/nwacus/avy/node_modules/jest-config/build/index.js:424:26)
    at async runCLI (/home/stevekuznetsov/code/nwacus/avy/src/github.com/nwacus/avy/node_modules/@jest/core/build/cli/index.js:151:59)
    at async Object.run (/home/stevekuznetsov/code/nwacus/avy/src/github.com/nwacus/avy/node_modules/jest-cli/build/run.js:130:37)
error Command failed with exit code 1.
```

This is with a freshly `rm -rf`-ed `node_modules` and full reinstall. Wondering if there's something else we're not correctly isolating from the environment?